### PR TITLE
Filter component updated

### PR DESCRIPTION
**Pull request by Smit2010**

### CODE REVIEW

**Comment by Smit2010** [comment]
@pranavpandey1998official Please review..

File: src/components/Search/Filter.js [hunk 1]
```diff
@@ -1,14 +1,220 @@
...
-		)
+		);
+	}
+};
+
+const Filter = withFormik({
+	handleSubmit(values, {props, resetForm}){
+		//handle form values here
+		console.log(values);
+		resetForm();
```
Comment (pranavpandey1998official) [inline src/components/Search/Filter.js]: remove this, we don't want that

Comment (Smit2010) [reply src/components/Search/Filter.js]: just empty withFormik()?

Comment (pranavpandey1998official) [reply src/components/Search/Filter.js]: nahi bas resetForm() remove karde

**Comment by Smit2010** [comment]
@pranavpandey1998official  Please review..

**Pull request merged**